client/src/main/java/io/prometheus/client/metrics/builtin/JVMMetrics.java

```diff
@@ -0,0 +1,233 @@
+package io.prometheus.client.metrics.builtin;
+
+import com.google.common.collect.ImmutableMap;
+import io.prometheus.client.Register;
+import io.prometheus.client.Registry;
+import static io.prometheus.client.Registry.emptyLabels;
+import io.prometheus.client.metrics.Gauge;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.management.JMException;
+import javax.management.MBeanServer;
+import javax.management.ObjectName;
+import java.lang.management.*;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Collection;
+import java.util.regex.Pattern;
+
+/**
+ * <p>
+ *   Standard JVM metrics via MXBeans. These are registered automatically.
+ * </p>
+ *
+ * @author kim.altintop@gmail.com (Kim Altintop)
+ */
+public class JVMMetrics {
+  private static final Logger log = LoggerFactory.getLogger(JVMMetrics.class);
+  private static final Pattern WHITESPACE = Pattern.compile("[\\s]+");
+  private static final Pattern CAMEL_CASE = Pattern.compile("([a-z\\d])([A-Z])");
+
+  @Register(name = "memory_usage_heap_total", 
```

Comment:
I've been implementing something similar. Can we prefix all jvm-related stats with jvm_?
jvm_gc_runs
jvm_buffer_pools?

This will make it easy to discover all jvm-related metrics in dropdowns and lists of collected metrics.